### ISSUE TITLE
Reindex unique constraint with INITIALLY DEFERRED options reset options

### ISSUE THREAD

**Issue by Napolskih** [grayhemp/pgtoolkit#10]
Hello

There is a table on the table to create a unique constraint with the options "initially deferred". When processing the table with the option reindex, the constraint reset option "initially deferred".

Constraint was originally created on the basis of the existing unique index using "add constraint using using index."

pgcompactor (PgToolkit) v1.0rc1

**Comment by grayhemp** [comment]
Right, this is a bug. Putting it down to my TODO list for the next version. Thanks for your feedback.

**Comment by Napolskih** [comment]
Maybe fix? :smile:

**Comment by isqad** [comment]
when?
